### PR TITLE
Simpify domain before parsing for EXPLAIN (TYPE IO)

### PR DESCRIPTION
**Pull request by shixuan-fan**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java [hunk 1]
```diff
@@ -522,7 +522,10 @@ else if (writerTarget instanceof CreateName || writerTarget instanceof InsertRef
             ImmutableSet.Builder<ColumnConstraint> columnConstraints = ImmutableSet.builder();
             for (Map.Entry<ColumnHandle, Domain> entry : constraint.getDomains().get().entrySet()) {
                 ColumnMetadata columnMetadata = metadata.getColumnMetadata(session, tableHandle, entry.getKey());
-                columnConstraints.add(new ColumnConstraint(columnMetadata.getName(), columnMetadata.getType().getTypeSignature(), parseDomain(entry.getValue())));
+                columnConstraints.add(new ColumnConstraint(
+                        columnMetadata.getName(),
+                        columnMetadata.getType().getTypeSignature(),
+                        parseDomain(entry.getValue().simplify())));
```
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java]: Consider adding a test and some explanation to the commit message.

**Comment by haozhun** [comment]
The change makes sense. I'll do a pass once you address comments from @findepi. I don't foresee any other issues.

File: presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java (hunk 1 above)
Comment (shixuan-fan) [reply presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java]: Done. Thanks for the suggestion!

**Comment by shixuan-fan** [comment]
Comment addressed.

**Pull request merged**